### ISSUE TITLE
certificate problem

### ISSUE THREAD

**Issue by squonk11** [DutchForeigner/signal-cli_rpi#6]
I am trying to use your signal-cli 0.9.0 on an RPI3 to register a phone number. Unfortunately I am getting the error message:
`Request verify error: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target`
It seems to be some issue with SSL certificate. Do you have a hint for me how to solve this?

**Comment by pmatuszy** [comment]
yeah. I had the same problem . use the latest build instead. I have a valid proc for rpi which took a while to develop

**Comment by squonk11** [comment]
Dear Paul,
thank you for your fast reply. Where can I find this latest build? Unfortunately I am not able to compile it on my RPi due to lack of available memory - it is a RPi3 with 1MB RAM and several services already installed so that the remaining memory is very low. Can you provide your build (.tar file?)? I am using RPi with `Linux jukebox 5.15.87-v7+ #1616 SMP Fri Jan 13 16:57:21 GMT 2023 armv7l GNU/Linux`, Buster, 64Bit.

**Comment by pmatuszy** [comment]
Hi - I put it here:
[https://github.com/pmatuszy/signal-cli-on-Raspberry-PI---WORKS-]

I did it a couple of times though on rpi4.... give it a try...

**Comment by squonk11** [comment]
I can't find a final .tar file there - it is just the steps to compile it, right?
I will try to do it on my rpi3 but currently it just has 236kB free RAM. Probably it will not work due to this.

**Comment by Nosnahc** [comment]
> Hi - I put it here: [https://github.com/pmatuszy/signal-cli-on-Raspberry-PI---WORKS-]
> 
> I did it a couple of times though on rpi4.... give it a try...

Hi,
don't work for me anymore.
Could you update it please?

Sorry I'm too noob to do it by myself, trying but not succesfull.
